app/src/main/java/com/dscvit/handly/ui/home/CollectionsFragment.kt

```diff
@@ -0,0 +1,227 @@
+package com.dscvit.handly.ui.home
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.recyclerview.widget.GridLayoutManager
+import com.dscvit.handly.R
+import com.dscvit.handly.adapter.CollectionsAdapter
+import com.dscvit.handly.model.Result
+import com.dscvit.handly.model.collection.CreateCollectionRequest
+import com.dscvit.handly.model.collection.DeleteCollectionRequest
+import com.dscvit.handly.model.collection.UpdateCollection
+import com.dscvit.handly.util.*
+import com.github.ybq.android.spinkit.style.Circle
+import com.github.ybq.android.spinkit.style.Wave
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import kotlinx.android.synthetic.main.add_collection_alert.view.*
+import kotlinx.android.synthetic.main.fragment_collections.*
+import kotlinx.android.synthetic.main.modify_collection_alert.view.*
+import org.koin.android.viewmodel.ext.android.sharedViewModel
+
+
+class CollectionsFragment : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_collections, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        collection_progress.hide()
+        collection_progress.setIndeterminateDrawable(Wave())
+
+        val homeViewModel by sharedViewModel<HomeViewModel>()
+
+        val collectionAdapter = CollectionsAdapter()
+        collection_recycler_view.apply {
+            layoutManager =
+                GridLayoutManager(requireContext(), 3, GridLayoutManager.VERTICAL, false)
+            adapter = collectionAdapter
+        }
+
+        getCollection(homeViewModel, collectionAdapter)
+
+        collection_recycler_view.addOnItemLongClickListener(object : OnItemClickListener {
+            override fun onItemClicked(position: Int, view: View) {
+                val dialogBuilder = MaterialAlertDialogBuilder(requireContext()).create()
+                val inflater = this@CollectionsFragment.layoutInflater
```

Comment:
I believe you can access this property directly by accessing the `layoutInflater` property of the current class. Consequently, there is no need to assign it to a variable.

Use the property directly everywhere.

Reply:
👍🏼